### DISCUSSION TITLE
SonarCloud Integration

### DISCUSSION THREAD

**Discussion by yasreebsaad** [DefectDojo/django-DefectDojo#6723] [Q&A] [answered]
Hi There,

I was looking for documentation to integrate SonarCloud with DefectDojo. I see we have something for SonarQueb but not for SonarCloud.

https://defectdojo.github.io/django-DefectDojo/integrations/parsers

SonarCloud is not supported as of now?

**Comment by manuel-sommer** [comment, marked as answer]
Hi @yasreebsaad,

sonarcloud integration is supported likewise to SonarQube. You have to add the organization ID also. See here:
https://defectdojo.github.io/django-DefectDojo/integrations/parsers/api/sonarqube/

I guess this can be marked as answered @mtesauro
